lib/screens/loading_info_screen.dart

```diff
@@ -93,16 +85,28 @@ class LoadingInfoScreenState extends State<LoadingInfoScreen> {
               if (_isDone)
                 ElevatedButton(
                   onPressed: () {
+                    AppStateHandler().setReadyState();
                     Navigator.pop(context);
                   },
                   child: const Text('Success'),
                 ),
-              if (widget.statusGreenNotifier.value == false)
+              if (context.watch<AppStateHandler>().syncState == SyncState.error)
```

Comment:
Der Error State bleibt rich beim nächsten Laden noch stehen, der Button ist somit schon während des Ladens zu sehen und ändert sich ggf auf Success
![Simulator Screenshot - iPhone 15 Pro Max - 2024-04-01 at 14 05 15](https://github.com/digital-scouts/dpsg-nami-app/assets/26873867/99996927-22a1-492a-beaf-492c837fd7ff)

Reply:
Fehler tritt weiterhin auf, wenn ein unbekannter Fehler aufgetreten ist.

1. Mit test Nutzer einloggen
2. In der Methode storeFakeSetOfMemberInHive einen throw error einbauen -> Hot reload
4. Über einstellungen Daten synchronisieren
5. Es wird geladen -> Fehler, du wirst ausgeloggt 
6. Login-Maske
7. throw error ausbauen -> hot reload
8. Mit test Nutzer einloggen 
9. direkt: syncState == SyncState.error

<img width="285" alt="Bildschirmfoto 2024-04-01 um 21 17 42" src="https://github.com/digital-scouts/dpsg-nami-app/assets/26873867/73a000ce-6703-4efe-b377-a8c1eba7e9b4">

Reply:
SessionExpired klappt

Reply:
Ich glaube das Problem lag woanders. Ich kann es nach [7b809d4](https://github.com/digital-scouts/dpsg-nami-app/pull/66/commits/7b809d45a1c9340e4e988895bdb75519d0134dec) nicht mehr reproduzieren.